osu.Game/Localisation/SongSelectStrings.cs

```diff
@@ -10,49 +10,49 @@ public static class SongSelectStrings
 
         /// <summary>
-        /// "Has been locally modified"
+        /// "Random"
         /// </summary>
-        public static LocalisableString LocallyModifiedTooltip => new TranslatableString(getKey(@"locally_modified_tooltip"), @"Has been locally modified");
+        public static LocalisableString Random => new TranslatableString(getKey(@"random"), @"Random");
 
         /// <summary>
-        /// "Manage collections"
+        /// "Rewind"
         /// </summary>
-        public static LocalisableString ManageCollections => new TranslatableString(getKey(@"manage_collections"), @"Manage collections");
+        public static LocalisableString Rewind => new TranslatableString(getKey(@"rewind"), @"Rewind");
 
         /// <summary>
-        /// "For all difficulties"
+        /// "Options"
         /// </summary>
-        public static LocalisableString ForAllDifficulties => new TranslatableString(getKey(@"for_all_difficulties"), @"For all difficulties");
+        public static LocalisableString Options => new TranslatableString(getKey(@"options"), @"Options");
 
         /// <summary>
-        /// "Delete beatmap"
+        /// "Local"
         /// </summary>
-        public static LocalisableString DeleteBeatmap => new TranslatableString(getKey(@"delete_beatmap"), @"Delete beatmap");
+        public static LocalisableString LocallyModified => new TranslatableString(getKey(@"locally_modified"), @"Local");
 
         /// <summary>
-        /// "For selected difficulty"
+        /// "Has been locally modified"
         /// </summary>
-        public static LocalisableString ForSelectedDifficulty => new TranslatableString(getKey(@"for_selected_difficulty"), @"For selected difficulty");
+        public static LocalisableString LocallyModifiedTooltip => new TranslatableString(getKey(@"locally_modified_tooltip"), @"Has been locally modified");
 
         /// <summary>
-        /// "Mark as played"
+        /// "Unknown"
         /// </summary>
-        public static LocalisableString MarkAsPlayed => new TranslatableString(getKey(@"mark_as_played"), @"Mark as played");
+        public static LocalisableString Unknown => new TranslatableString(getKey(@"unknown"), @"Unknown");
 
         /// <summary>
-        /// "Clear all local scores"
+        /// "Total Plays"
         /// </summary>
-        public static LocalisableString ClearAllLocalScores => new TranslatableString(getKey(@"clear_all_local_scores"), @"Clear all local scores");
+        public static LocalisableString TotalPlays => new TranslatableString(getKey(@"total_plays"), @"Total Plays");
 
         /// <summary>
-        /// "Edit beatmap"
+        /// "Personal Plays"
         /// </summary>
-        public static LocalisableString EditBeatmap => new TranslatableString(getKey(@"edit_beatmap"), @"Edit beatmap");
+        public static LocalisableString PersonalPlays => new TranslatableString(getKey(@"personal_lays"), @"Personal Plays");
```

Comment:
Maybe just me, but I think you've made a typo down here
```suggestion
        public static LocalisableString PersonalPlays => new TranslatableString(getKey(@"personal_plays"), @"Personal Plays");
```

Reply:
yeah, TY, I'm definitely need to find a way to add these strings not manually